src/libraries/Microsoft.Extensions.DependencyInjection/tests/Microsoft.Extensions.DependencyInjection.Tests.csproj

```diff
@@ -2,18 +2,20 @@
 
   <PropertyGroup>
     <TargetFrameworks>$(NetCoreAppCurrent);$(NetFrameworkCurrent)</TargetFrameworks>
-    <RootNamespace>Microsoft.Extensions.DependencyInjection</RootNamespace>
+    <DefineConstants>$(DefineConstants);SKIP_FAILING</DefineConstants>
   </PropertyGroup>
 
   <ItemGroup>
-    <Reference Include="Microsoft.Extensions.DependencyInjection.Abstractions" />
-    <Reference Include="Microsoft.Extensions.DependencyInjection.Specification.Tests" />
-    <Reference Include="Microsoft.Extensions.DependencyInjection" />
-    <Reference Include="Microsoft.Extensions.Logging.Abstractions" />
-    <Reference Include="Newtonsoft.Json" />
+    <Compile Include="*.cs" />
```

Comment:
Is there a reason why we aren't just including all source recursively?  `EnableDefaultItems`?

If there is source in this directory that isn't needed, just delete it.